### ISSUE TITLE
Duplicate image on paste

### ISSUE THREAD

**Issue by Eraldo** [chenjuneking/quill-image-drop-and-paste#52]
I have defined a custom image handler and that does trigger a modal to upload an image and that does work.
I also replaced it with a function that only does `alert("Works!");` just to be sure it's not the cause of my issue.

However immediately upon pasting the image inside the editor, I immediately get an inserted image as a string blob inserted into the editor. This way after continuing with my upload modal, I end up having two images.
Once the inline string blob and once an image tag with my uploaded image as src.

PS: There is a typo in the docs: "didnot" => "did not". ;)
```
If you didnot config a image handler, it will insert the image with dataURL into the quill editor directory after your drop/paste.
```

Any help on how to prevent the default behaviour is highly appreciated. 🙏

**Comment by chenjuneking** [comment]
Sorry for your trouble, could you provide some codes or screen videos showing the issue?

**Comment by Eraldo** [comment]
> Sorry for your trouble, could you provide some codes or screen videos showing the issue?

Sure, thanks for asking: https://youtu.be/245KPVeXZ3g

**Comment by linxianxi** [comment]
same

**Comment by dalexander-trc** [comment]
Also experiencing this.
```json   
"ngx-quill": "~25.3.0",
"parchment": "~3.0.0",
"quill": "~2.0.2",
"quill-image-drop-and-paste": "~1.3.0"
```

**Comment by granawkins** [comment]
same

**Comment by VaguelySerious** [comment]
Hi all, it looks like Quill `>2.0.0` might have added native image upload functionality which conflicts with this module, hence duplicate uploads. If you want to fix the duplicate issue easily, just add `uploader: { handler: () => {} },` to your `modules` list in Quill, or do something like this:
```js
class NoopClass {}

Quill.register({
  "modules/uploader": NoopClass
}, true);
```
I don't know if the native version is better or worse, just that this fixes it. Cheers.

**Comment by chenjuneking** [comment]
fixed on version `^2.0.0`.
